### PR TITLE
Nodejs week3 homework

### PR DESCRIPTION
**Pull request by na-talia**

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -104,6 +107,7 @@ app.get("/last-meal", async (req, res) => {
   }
 });
 
+
```
Comment (YurijPaharev) [inline src/backend/app.js:110]: Try not to leave any trialing spaces

File: src/backend/api/meals.js [hunk 2]
```diff
@@ -2,6 +2,112 @@ const express = require("express");
...
+  // Returns all meals sorted by the given key. (api/meals?sortKey=price)
+  // Returns all meals sorted in the given direction. (api/meals?sortKey=price&sortDir=desc)
+
+  const sortedByKey = req.query.sortKey;
+  const sortedByDirection = req.query.sortDir;
+  if (
+    sortedByKey === "when" ||
+    sortedByKey === "max_reservations" ||
+    sortedByKey === "price"
+  ) {
```
Comment (YurijPaharev) [inline src/backend/api/meals.js]: you can group the expected results in an array and then check if sortedByKey is in them, like [a, b, c].includes(sortedByKey)

**Review by YurijPaharev — APPROVED** [review]
Well done! Please check small comments in the code

**Comment by na-talia** [comment]
> Well done! Please check small comments in the code

Thank you for the review!

**Pull request merged**